tdrs-backend/tdpservice/reports/views.py

```diff
@@ -22,6 +29,16 @@ class ReportFileViewSet(ModelViewSet):
     serializer_class = ReportFileSerializer
     queryset = ReportFile.objects.all()
 
+    @action(methods=["get"], detail=True)
+    def download(self, request, pk=None):
+        """Retrieve a file from s3 then stream it to the client."""
+        record = DownloadReportFileSerializer(ReportFile.objects.get(id=pk))
```

Comment:
We don't actually need the serializer here, since this is a detail endpoint we can use the DRF provided method to get the relevant ReportFile:
```suggestion
        record = self.get_object()
```